config/puma.rb

```diff
@@ -30,18 +30,27 @@
 # you need to make sure to reconnect any threads in the `on_worker_boot`
 # block.
 #
-preload_app!
+# preload_app!
```

Comment:
The upgrade task suggested removing this, and the comments here suggest we need to do some other config for this to work correctly (which we had commented out).  Rails 5.2 separately adds [bootsnap](http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#bootsnap), so I think better to look at this separately.